Drive/main.cpp

```diff
@@ -0,0 +1,25 @@
+#include "./RoverDriveSystems.hpp"
+
+int main()
+{
+  string mode;
+  float speed;
+  units::angle::degree_t angle;
+
+  RoverDriveSystems roverDriver;
+  roverDriver.Initialize();
+
+      while (true)
+  {
+    // {speed, angle, mode } = getMissionControlCommands()
```

Comment:
```C++
struct DriveCommand_t
{
  bool is_operational = false;
  char mode = 'S';
  float speed = 0;
  units::angle::degree_t angle = 0_deg;
};

DriveCommand_t GetMissionControlCommands() {
  return DriveCommand_t{};
}
```

Actually add in the function so that this can compile.